src/lesson-5-1-demo/DemoClientSidePagination.tsx

```diff
@@ -64,26 +74,30 @@ export const UsersListWithSorting: React.FC<UsersListProps> = ({
   const [sortDirection, setSortDirection] = useState<SortDirection>("ASC");
 
   const handlePreviousClick = () => {
-    // TODO
+    setCursor(cursor - pageSize);
   };
   const handleNextClick = () => {
-    //TODO
+    setCursor(cursor + pageSize);
```

Comment:
imagine we have the following array and page configuration:
```javascript
let cursor = 5;
let pageSize = 5;
const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
//               ^              ^              ^
//               prev page      current page   next page
```

to get to the next page, we set increment the cursor by the page size
to get to the previous page, we decrement the cursor by the page size